lightning-types/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "lightning-types"
+version = "0.1.0"
+authors = ["Matt Corallo"]
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/lightningdevkit/rust-lightning/"
+description = """
+Basic types which are used in the lightning network
+"""
+edition = "2021"
+
+[package.metadata.docs.rs]
+rustdoc-args = ["--cfg", "docsrs"]
+
+[features]
```

When building `lightning-types` by itself, I found that the std/no-std features are missing:

```suggestion
[features]
default = ["std"]
no-std = ["bitcoin/no-std"]
std = ["bitcoin/std", "bech32/std"]
```